blocks/identity-block/intl.json

```diff
@@ -2078,5 +2078,26 @@
 		"vi": "Tên người dùng",
 		"zh-CN": "用户名",
 		"zh-TW": "使用者名稱"
+	},
+	"identity-block.connect-identity": {
+		"en": "Connect"
+	},
+	"identity-block.connected-identity": {
+		"en": "Connected"
+	},
+	"identity-block.login-form-error.captcha-token-required": {
+		"en": "Captcha token required."
+	},
+	"identity-block.social-google-one-tap-notification": {
+		"en": "Sign-in prompt was suppressed by the user or dismissed. Please try again later or use another sign-in method."
+	},
+	"identity-block.social-signOn-unlink-error": {
+		"en": "At least one identity needs to remain on the user"
+	},
+	"identity-block.social-signOn-facebook-login": {
+		"en": "Sign in with Facebook"
+	},
+	"identity-block.social-signOn-facebook-signUp": {
+		"en": "Sign up with Facebook"
```

Comment:
@LauraPinilla Can you please add other languages translations as well here?

Reply:
@malavikakoppula for now I added the same English key for all languages, only Irish News is using our blocks, and they are using English. There are few words to create a new order, or we'll need to pay the minimum price per language. 

<img width="1487" alt="Screenshot 2024-07-26 at 5 04 04 p m" src="https://github.com/user-attachments/assets/113085db-5ae0-4e22-8e3d-0a00dbc3fdce">